index.js

```diff
@@ -1,17 +1,17 @@
 'use strict';
 var relative = require('path').relative;
-var hasFlag = require('has-flag');
 var chalk = require('chalk');
 var serializeError = require('serialize-error');
+var babel = require('./lib/babel');
 var globals = require('./lib/globals');
 var Runner = require('./lib/runner');
 var send = require('./lib/send');
 var log = require('./lib/logger');
 
-var runner = new Runner();
-
 // note that test files have require('ava')
-require('./lib/babel').avaRequired = true;
+babel.avaRequired = true;
+
+var runner = new Runner(babel.opts);
```

Comment:
It's weird that `babel` exports the options. I think we should just get them directly here too.